python/langsmith/evaluation/evaluator.py

```diff
@@ -263,13 +263,63 @@ def _coerce_evaluation_result(
                     "Expected an EvaluationResult object, or dict with a metric"
                     f" 'key' and optional 'score'; got empty result: {result}"
                 )
+            if (
+                len(set(result) & {"comment", "justification", "reasoning", "reason"})
```

Comment:
Why so many aliases? Agree `comment` isn't the best but we should pick one alternative and stick with it no?

Reply:
I mean why stop at 4 then?